src/composables/useClient.ts

```diff
@@ -10,6 +10,22 @@ export function useClient() {
   const auth = getInstance();
   const { notify } = useNotifications();
 
+  function successNotification(keyword: string) {
+    let notification;
+
+    if (keyword === 'delete-proposal')
+      notification = ['red', t('notify.proposalDeleted')];
```

Comment:
This should be green
```suggestion
      notification = ['green', t('notify.proposalDeleted')];
```